Extension/Xamarin-Extension/Create-Project.md

```diff
@@ -22,26 +22,26 @@ To create the **Syncfusion Xamarin Application** in Visual Studio 2017, follow t
 	**Option 1:**  
 	Click **Syncfusion Menu** and choose **Essential Studio for Xamarin > Create New Syncfusion Project…** in **Visual Studio**.
 
-	![Choose Syncfusion Xamarin application from Visual Studio new project dialog via Syncfusion menu](Syncfusion-Project-Templates_images/Syncfusion_Menu_ProjectTemplate.png)
+	![Choose Syncfusion Xamarin Extension application from Visual Studio new project dialog via Syncfusion menu](syncfusion-project-templates_images/xamarin-extension-create-project-dialog-project-application.png)
```

Comment:
Don't need to add Extension in Image ALT text, revert in all places both image name and alt text.

Reply:
Updated.